gcloud-java-storage/src/main/java/com/google/gcloud/storage/Bucket.java

```diff
@@ -654,16 +651,51 @@ public Blob create(String blob, byte[] content, String contentType, BlobTargetOp
    *
    * @param blob a blob name
    * @param content the blob content as a stream
-   * @param contentType the blob content type. If {@code null} then
-   *     {@value com.google.gcloud.storage.Storage#DEFAULT_CONTENT_TYPE} is used.
+   * @param contentType the blob content type
    * @param options options for blob creation
    * @return a complete blob information
    * @throws StorageException upon failure
    */
   public Blob create(String blob, InputStream content, String contentType,
       BlobWriteOption... options) {
-    BlobInfo blobInfo = BlobInfo.builder(BlobId.of(name(), blob))
-        .contentType(MoreObjects.firstNonNull(contentType, Storage.DEFAULT_CONTENT_TYPE)).build();
+    BlobInfo blobInfo = BlobInfo.builder(BlobId.of(name(), blob)).contentType(contentType).build();
+    StorageRpc.Tuple<BlobInfo, Storage.BlobWriteOption[]> write =
+        BlobWriteOption.toWriteOptions(blobInfo, options);
+    return storage.create(write.x(), content, write.y());
+  }
+
+  /**
+   * Creates a new blob in this bucket. Direct upload is used to upload {@code content}.
+   * For large content, {@link Blob#writer(com.google.gcloud.storage.Storage.BlobWriteOption...)}
+   * is recommended as it uses resumable upload. MD5 and CRC32C hashes of {@code content} are
+   * computed and used for validating transferred data.
+   *
+   * @param blob a blob name
+   * @param content the blob content
+   * @param options options for blob creation
+   * @return a complete blob information
+   * @throws StorageException upon failure
+   */
+  public Blob create(String blob, byte[] content, BlobTargetOption... options) {
+    BlobInfo blobInfo = BlobInfo.builder(BlobId.of(name(), blob)).build();
```

Comment:
Oh, I see. Yes there is a conceptual difference between unset and explicit set to null.
However, because this is create and not an update is there a practical difference?